dispersal/settings.py

```diff
@@ -158,13 +150,15 @@
 # TODO ADD back basic auth eventually.
 REST_FRAMEWORK = {
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        # 'rest_framework.authentication.BasicAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.SessionAuthentication',
     ),
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
     'PAGE_SIZE': 50
 }
 
+SITE_URL = os.environ.get('SITE_URL', 'http://127.0.0.1:8000/')
```

Comment:
does this need changed for the heroku deployment

Reply:
Heroku has a environment variable set

Reply:
127 etc.. is the default value if the request to get environ variable fails :) aka it'll be good